infra/command/format

```diff
@@ -124,16 +123,9 @@ function check_cpp_files() {
 
   # Check c++ files: replace ' ' with newline, check with grep
   FILES_TO_CHECK_CPP=`echo "$FILES_TO_CHECK" | tr ' ' '\n' | egrep '((\.c[cl]?)|(\.cpp)|(\.h(pp)?))$'`
-  # Manually ignore style checking
-  FILES_TO_CHECK_CPP=`echo "$FILES_TO_CHECK_CPP" | egrep -v '((/NeuralNetworks\.h)|(/NeuralNetworksExtensions\.h))$'`
```

Comment:
Q) are these related with `.FORMATDENY` ? or another format update ? (I assume we don't need to check these `NeuralNetworks` files)

Reply:
They are ignored by `compiler/ann-api/include/.clang-format` and `tests/nnapi/bridge/include/.clang-format`. I'll update commit comment.